crates/medmodels-core/src/medrecord/querying/edges/operand.rs

```diff
@@ -318,7 +318,7 @@ impl DeepClone for EdgeIndicesComparisonOperand {
     fn deep_clone(&self) -> Self {
         match self {
             Self::Operand(operand) => Self::Operand(operand.deep_clone()),
-            Self::Indices(value) => Self::Indices(value.clone()),
+            Self::Indices(indices) => Self::Indices(indices.clone()),
         }
```

Comment:
because there is a (optional) convention for the argument in the `from` function to be called `value`. but yeah it's inconsistent (in the module). I'll update